### PR TITLE
fix(IAM Policy Management): re-gen service and add examples after recent API changes

### PR DESCRIPTION
**Pull request by padamstx**
## PR summary
Updates the IAM Policy Management service and adds examples after recent API changes.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by padamstx** [comment]
@guicabr, this is the Python PR.  The examples are in the "example" directory, similar to the node sdk project.

**Review by padamstx — COMMENTED** [review]
Looks good.  I made one minor suggestion about variable names.

File: examples/test_iam_policy_management_v1_examples.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+                tags=[resource_tag])
+
+            policy = iam_policy_management_service.create_policy(
+                type='access',
+                subjects=[policy_subject],
+                roles=[policy_role],
+                resources=[policy_resource]
+            ).get_result()
+
+            print(policy)
```
Comment (padamstx) [inline examples/test_iam_policy_management_v1_examples.py:120]:
```suggestion
                print(json.dumps(policy, indent=2))

```

**Review by guicabr — APPROVED** [review]
LGTM

**Pull request merged**